src/main/java/de/hysky/skyblocker/skyblock/garden/VisitorHelper.java

```diff
@@ -69,19 +69,25 @@ public static void onMouseClicked(double mouseX, double mouseY, int mouseButton,
 
             for (Object2IntMap.Entry<String> itemEntry : visitorEntry.getValue().object2IntEntrySet()) {
                 String itemText = itemEntry.getKey();
-                textWidth = textRenderer.getWidth(itemText + " x" + itemEntry.getIntValue());
+                textWidth = textRenderer.getWidth(itemEntry.getKey() + " x" + itemEntry.getIntValue());
+
+                // Calculate the x position for the "Copy Amount" text
+                int visitorNameWidth = textRenderer.getWidth(visitorEntry.getKey().left());
+                int copyAmountTextX = TEXT_START_X + visitorNameWidth + 7;
 
                 if (isMouseOverText(mouseX, mouseY, TEXT_START_X + TEXT_INDENT, yPosition, textWidth, textHeight)) {
                     MessageScheduler.INSTANCE.sendMessageAfterCooldown("/bz " + itemText);
+                }
 
+                if (isMouseOverText(mouseX, mouseY, copyAmountTextX, yPosition, textRenderer.getWidth("[Copy Amount]"), textHeight)) {
                     MinecraftClient.getInstance().keyboard.setClipboard(String.valueOf(itemEntry.getIntValue()));
                     return;
                 }
```

Comment:
Change the x to not overlap and add some feedback to the player. You will have to import some stuff.
```suggestion
                MinecraftClient client = MinecraftClient.getInstance();
                if (client.player != null && isMouseOverText(mouseX, mouseY, TEXT_START_X + TEXT_INDENT + textWidth, yPosition, textRenderer.getWidth(" [Copy Amount]"), textHeight)) {
                    client.keyboard.setClipboard(String.valueOf(itemEntry.getIntValue()));
                    client.player.sendMessage(Constants.PREFIX.get().append("Copied amount successfully"));
                    return;
                }
```

Reply:
gotta fix the clipboard, doesn't wanna copy with this change :/